### PR TITLE
Update G-Cloud 8 welcome email content

### PR DESCRIPTION
**Pull request by TheDoubleK**
Text taken from this doc: https://docs.google.com/document/d/1rGlCOCVoTU7ryLY0hew4ZQl58_uHMGBH1aj1ko8cAZg/edit?ts=571e1601

New version looks like this when it gets sent:
screen shot 2016-05-17 at 12 07 04

### CODE REVIEW

File: app/templates/emails/g-cloud-8_application_started.html [hunk 1]
```diff
@@ -3,25 +3,49 @@
...
+      </li>
+      <li>
+          find out what G8 means for you if you’re a G-Cloud 6 or G-Cloud 7 supplier:<br />
+          <a href="https://digitalmarketplace.blog.gov.uk/2016/04/18/what-g-cloud-8-means-for-g-cloud-6-and-g-cloud-7-suppliers/">
+              https://digitalmarketplace.blog.gov.uk/2016/04/18/what-g-cloud-8-means-for-g-cloud-6-and-g-cloud-7-suppliers
+          </a>
+      </li>
+      <li>
+          see the latest timetable of important dates:<br />
+          <a href="https://digitalmarketplace.blog.gov.uk/2016/05/17/g-cloud-8-apply-to-supply-your-cloud-services/">
```
Comment (pcraig3) [inline app/templates/emails/g-cloud-8_application_started.html]: This link goes live later today, right?

Comment (TheDoubleK) [inline app/templates/emails/g-cloud-8_application_started.html]: Yep.  Gemma is going to publish it just as the thing goes live.

Comment (pcraig3) [inline app/templates/emails/g-cloud-8_application_started.html]: Right, that's good then.

**Comment by pcraig3** [comment]
👍  ✉️  👍

**Pull request merged**